src/client/dfs/dfs.c

```diff
@@ -1255,6 +1255,12 @@ dfs_get_sb_layout(daos_key_t *dkey, daos_iod_t *iods[], int *akey_count,
 	return 0;
 }
 
+void
+dfs_free_sb_layout(daos_iod_t *iods[])
+{
+	D_FREE(*iods);
+}
+
```

Comment:
https://github.com/daos-stack/daos/pull/8207 re-adds this, but on a different line. It should be removed here

Reply:
Just revert my change in this file.